code/modules/events/blob.dm

```diff
@@ -4,7 +4,7 @@
 	weight = 10
 	max_occurrences = 1
 
-	min_players = 20
+	min_players = 30
```

Comment:
I would  not do a drastic change to 30. How about 25 instead

Reply:
We could try 25 and see how it goes

Reply:
please reflect that you changed it to 25 in the descrption

Reply:
please reflect that you changed it to 25 in the descrption